### ISSUE TITLE
More meaningful error messages

### ISSUE THREAD

**Issue by fcin** [angular/protractor#4002]
Hi,
It would be great if messages had more information about the error. Sometimes I get message like this one: 
`Failed: element not visible (Session info: chrome=55.0.2883.87) (Driver info: chromedriver=2.25.426923 (0390b88869384d6eb0d5d09729679f934aab9eed),platform=Windows NT 10.0.10240 x86_64) (WARNING: The server did not provide any stacktrace information) Command duration or timeout: 20 milliseconds`

It would be nice, if the message would contain at least a name of the element or name of a locator that is not visible, because looking only at the message and not stacktrace makes it difficult to diagnose which element in test causes the problem. It is even more difficult in projects that use typescript, because stacktrace and line with error refers to `.js` file that is usually a little bit different than `.ts`, so the error line and `.ts` test line don't match.

**Comment by inad9300** [comment]
Please!

**Comment by rooby** [comment]
Yes, these type of messages, like " Failed: element not interactable" are almost pointless without even a line number or anything to go off.
